pkg/test/utils/testing.go

```diff
@@ -15,10 +15,12 @@ import (
 // If testToRun is set to a non-empty string, it is passed as a `-run` argument to the go test harness.
 func RunTests(testName string, testToRun string, testFunc func(*testing.T)) {
 	// Set the verbose test flag to true since we are not using the regular go test CLI.
-	flag.Lookup("test.v").Value.Set("true")
+	flag.Set("test.v", "true")
```

Comment:
why the switch here?

Reply:
I just noticed when I opened the docs that they had a helper method for it already, so cleaned it up